react/ActionMenu/index.spec.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
+import ActionMenu, { ActionMenuItem } from './'
+
+describe('ActionMenu', () => {
+  let createRangeBackup
+  beforeAll(() => {
+    createRangeBackup = global.document.createRange
+    global.document.createRange = jest.fn(() => ({
+      setStart: () => {},
+      setEnd: () => {},
+      commonAncestorContainer: {
+        nodeName: 'BODY',
+        ownerDocument: document
+      }
+    }))
```

Comment:
Why is this needed ?

Reply:
We have issue for testing component using Tooltip from MUI. See https://github.com/mui-org/material-ui/issues/15726#issuecomment-493124813 

jsdom-16 fix the issue, but we need to use https://www.npmjs.com/package/jest-environment-jsdom-sixteen. I tried few months ago but still the issue. Maybe it's fine now.

Reply:
AMHA this should not be in this test file and should be called with a name that reviewers can understand.

Tooltip-testing.js

```
const fixTooltipTesting = () => {
  let createRangeBackup
  beforeAll(() => {
    createRangeBackup = global.document.createRange
    global.document.createRange = jest.fn(() => ({
      setStart: () => {},
      setEnd: () => {},
      commonAncestorContainer: {
        nodeName: 'BODY',
        ownerDocument: document
      }
    }))

  afterAll(() => {
    global.document.createRange = createRangeBackup
  })
}
```

This way we do not need a comment, it's clearer for the reviewer.

Reply:
and we can import it from other libs 👍